code/modules/research/designs/autolathe_designs.dm

```diff
@@ -872,14 +872,6 @@
 	build_path = /obj/item/assembly/mousetrap
 	category = list("initial", "Miscellaneous")
 
-/datum/design/videocam
-	name = "Video Camera"
-	id = "videocam"
-	build_type = AUTOLATHE
-	materials = list(MAT_METAL = 120, MAT_GLASS = 60)
-	build_path = /obj/item/videocam
-	category = list("initial", "Miscellaneous")
-
```

Comment:
Possibly comment this out incase it needs to be easily added back in the future for whatever reason, but at the same time its only 7 lines.